### ISSUE TITLE
cannot find any release URL for slug ID: vmware-tanzu-greenplum

### ISSUE THREAD

**Issue by chewygranola** [greenplum-db/go-gpdb#51]
Unable to download, slug ID changed.

**Comment by chewygranola** [comment]
```
DEBU[2023-03-28 11:47:51] Reading the configuration file and loading to the memory: /home/gpadmin/config.yml  file="config.go:142"
DEBU[2023-03-28 11:47:51] Checking if the directories needed for the program exists  file="config.go:82"
DEBU[2023-03-28 11:47:51] Checking if the directory /usr/local/src/gpdbinstall/temp/ exists  file="helpers.go:39"
DEBU[2023-03-28 11:47:51] Checking if the directory /usr/local/src/gpdbinstall/download/ exists  file="helpers.go:39"
DEBU[2023-03-28 11:47:51] Checking if the directory /usr/local/src/gpdbinstall/env/ exists  file="helpers.go:39"
DEBU[2023-03-28 11:47:51] Checking if the directory /usr/local/src/gpdbinstall/uninstall/ exists  file="helpers.go:39"
DEBU[2023-03-28 11:47:51] Checking if the directory /usr/local/src/gpdbinstall/future_reference/ exists  file="helpers.go:39"
INFO[2023-03-28 11:47:51] Starting the program to download the product  file="download.go:456"
INFO[2023-03-28 11:47:51] Getting the access token from the UAA token provided.  file="request.go:26"
INFO[2023-03-28 11:47:51] Obtaining the product ID                      file="download.go:270"
DEBU[2023-03-28 11:47:51] Requesting data from the url: https://network.pivotal.io/api/v2/products  file="request.go:100"
INFO[2023-03-28 11:47:52] Obtaining the releases for product:           file="download.go:354"
FATA[2023-03-28 11:47:52] cannot find any release URL for slug ID: vmware-tanzu-greenplum  file="download.go:369"
```

**Comment by chewygranola** [comment]
Fixed by #52 .
